src/library/Button/__tests__/Button.spec.js

```diff
@@ -29,5 +44,87 @@ describe('Button', () => {
     expect(button.props().onClick).toHaveBeenCalled();
   });
 
+  describe('types', () => {
+    describe("when it's a link", () => {
+      it(`<a> type="button" should be filtered`, () => {
```

Comment:
While I know it is relatively popular in the Ruby testing world, "should" is a superfluous word and is better avoided in "it" blocks.  That is the convention of this project anyway.  Please just use direct phrasing.  e.g. "button type is filtered", "video type is not filtered", etc.